api/models/adoptionVisit.js

```diff
@@ -21,20 +21,20 @@ const adoptionVisitSchema = new mongoose.Schema({
         max: 120,
         required: true
     },
-    animalID : {
-        type: mongoose.Schema.Types.ObjectId,
-        ref: 'Animal',
+    isVisitDone: {
+        type: Boolean,
+        required: true,
+        default: false
```

Comment:
tutaj chyba required nie jest potrzebne. Jeśli masz default to w zapytaniu post automatycznie zostanie ta wartość dodana.

Reply:
Już zaktualizowane :)